lib/hanami/assets/helpers.rb

```diff
@@ -682,6 +752,15 @@ def _typed_asset_path(source, ext)
         asset_path(source)
       end
 
+      def _sri?
+        !!self.class.assets_configuration.sri
+      end
+
+      def _sri_value(source, ext)
+        source = "#{ source }#{ ext }" unless source.match(/#{ Regexp.escape(ext) }\z/)
+        self.class.assets_configuration.sri_value(source) unless _absolute_url?(source)
```

Comment:
@jodosha I looked at how SRI was being used already, and a number of CDN hosts are using multiple SRI digest hash algorithms. Case in point, BootstrapCDN:

```html
<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-7s5uDGW3AHqw6xtJmNNtr+OBRJUlgkNJEo78P4b0yRw= sha512-nNo+yCHEyn0smMxSswnf/OnX6/KwJuZTlNZBjauKhTK0c+zT+q5JOCx0UFhXQ6rJR9jg6Es8gPuD2uZcYDLqSw==" crossorigin="anonymous”>
```

I would prefer `subresource_integrity` as the option, so will go with that.